lib/dust.js

```diff
@@ -849,7 +849,10 @@
       SQUOT  = /\'/g;
 
   dust.escapeHtml = function(s) {
-    if (typeof s === 'string') {
+    if (typeof s === "string" || (s && s.toString)) {
```

Comment:
would it be too paranoid to do:

`typeof s === 'object' && Array.prototype.toString.call(s)`

I'm not sure what happens if s is an object with the key toString or a function literal... it's a similar problem to `context = [] ` and we try to reference `context.sort` . mildly related to #469